dali/kernels/slice/slice_kernel_test.h

```diff
@@ -346,6 +338,12 @@ using SLICE_TEST_TYPES_CPU_ONLY = ::testing::Types<
     SliceTestArgs<int, int, 3, 1, 20, ArgsGen_PadAlsoChDim_ChFirst<int, 3>, 3, 20, 20>
 >;
 
+using SLICE_TEST_TYPES_CPU_ONLY = ::testing::Types<
```

Comment:
We support this only for CPU while not for the GPU?

Reply:
I think it was hard to generate the float16 data in the GPU, if I remember correctly. I could give it another try

Reply:
Basically the test utilities to generate, transfer to GPU and compare data can't deal with float16. It'd be an effort on its own to make it happen